### PR TITLE
Provide a better diagnosis of 403 errors

### PR DESCRIPTION
**Pull request by mbland**
I'll push the updated nginx config to production once this propagates to the Hub.

cc: @afeld @wslack

### CODE REVIEW

**Comment by wslack** [comment]
Thank you Mike!

**Comment by mbland** [comment]
Actually, I think I'll need to update the oauth2_proxy config for this to really take effect; will figure that out shortly, but in a meeting now.

File: pages/403.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+layout: default
+permalink: /403/
+title: Unauthorized
+---
+<section class="fours">
+  <div class="container">
+    <div class="fours-content">
+      <h1>403</h1>
+      <h2><em>&quot;Who's there?&quot;</em><br/>
+        William Shakespeare, <em>The Tragedy of Hamlet, Prince of Demark</em></h2>
```
Comment (afeld) [inline pages/403.html]: Haha what??

File: pages/403.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="container">
+  <div class="fours-content">
+    <p>It seems you're not in our access list, or you may have accidentally
+    used the wrong account to authenticate with MyUSA. If so you can
+    <a href="https://staging.my.usa.gov/">return to MyUSA to log out</a>, then
```
Comment (afeld) [inline pages/403.html]: Not https://alpha.my.usa.gov?

Comment (wslack) [inline pages/403.html]: Nope, it's staging.

**Pull request merged**